Lib/multiprocessing/managers.py

```diff
@@ -511,6 +511,7 @@ def connect(self):
         Listener, Client = listener_client[self._serializer]
         conn = Client(self._address, authkey=self._authkey)
         dispatch(conn, None, 'dummy')
+        BaseProxy._address_to_local.pop(self._address, None)
```

Comment:
i'm following line 664 here, but maybe it should be
`del BaseProxy._address_to_local[self._address][0].connection`